lib/auth/init.go

```diff
@@ -342,6 +324,30 @@ func Init(cfg InitConfig, seedConfig bool) (*AuthServer, *Identity, error) {
 	return asrv, identity, nil
 }
 
+func migrateCertAuthority(asrv *AuthServer, in services.CertAuthority) error {
+	raw, ok := (in.GetRawObject()).(services.CertAuthorityV1)
+	if !ok {
+		return nil
+	}
+	_, err := asrv.GetRole(services.RoleNameForCertAuthority(in.GetClusterName()))
+	if err == nil {
+		return nil
+	}
+	if !trace.IsNotFound(err) {
```

Comment:
we enforce the behavior in backend acceptance tests:

https://github.com/gravitational/teleport/blob/master/lib/services/suite/suite.go#L464